### PR TITLE
Disable using QEMU session on CentOS 8

### PR DESCRIPTION
**Pull request by anoopcs9**

### CODE REVIEW

**Comment by spuiuk** [comment]
Why not set default behaviour to use this setting and not add it as an exception for CentOS7? CentOs 8 and Fedora should be fine with this setting.

**Comment by anoopcs9** [comment]
> Why not set default behaviour to use this setting and not add it as an exception for CentOS7? CentOs 8 and Fedora should be fine with this setting.

`libvirt.use_qemu_session` is enabled by default. So I thought it would be good to disable it in special cases only.

**Comment by spuiuk** [comment]
> > Why not set default behaviour to use this setting and not add it as an exception for CentOS7? CentOs 8 and Fedora should be fine with this setting.
> 
> `libvirt.use_qemu_session` is enabled by default. So I thought it would be good to disable it in special cases only.

(We discussed this at the call today. Adding it here too.)
We will need this for all future versions of host environment which includes CentOS8, Fedora.. etc. 
So in my opinion, it would be better to add this directive by default and disable it only if it is running on CentOS 7.

File: vagrant/templates/Vagrantfile.j2 [hunk 1]
```diff
@@ -19,19 +22,12 @@ Vagrant.configure("2") do |config|
...
-        #   config.vm.provider :libvirt do |libvirt|
-        #     libvirt.qemu_use_session = false
-        #   end
-        # end
-        # ```
-        #v.qemu_use_session = false
+        # We can not use qemu_use_session universally. See section "Upgrade/compatibility impact"
+        # from the following Fedora doc:
+        # https://fedoraproject.org/wiki/Changes/Vagrant_2.2_with_QEMU_Session
+        if HOST_OS != "CentOS" || HOST_OS_VERS != "7"
```
[the remark this answers is not in the document]
Comment (anoopcs9) [reply vagrant/templates/Vagrantfile.j2]: Done

**Review by spuiuk — APPROVED** [review]
Looks good. ACK

**Pull request merged**